pkg/systemd/logDetails.jsx

```diff
@@ -148,7 +148,7 @@
                     } else
                         this.setState({ entry: null, loading: false, error: _("Journal entry not found") });
                 })
-                .catch(error => this.setState({ entry: null, loading: false, error: error }));
+                .catch(error => this.setState({ entry: null, loading: false, error }));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)